src/main/daml/ContingentClaims/Core/Builders.daml

```diff
@@ -42,16 +42,19 @@ floating principal coupon asset fixings =
 fixed : forall t x ccy o. x -> x -> ccy -> [t] -> Claim t x ccy o
 fixed principal coupon = floating (O.pure principal) (O.pure coupon)
 
--- | European option on the passed claim. e.g. call option on S&P 500:
+-- | European option on the input claim. At maturity, the holder must `EXERCISE` or `EXPIRE`
+-- the claim. e.g. call option on S&P 500:
 -- ```
 -- european (date 2021 05 14) (observe "SPX" - pure 4200)
 -- ```
 european : t -> Claim t x a o -> Claim t x a o
 european maturity payoff = when (at maturity) (("EXERCISE", payoff) `or` ("EXPIRE", zero))
 
--- | Bermudan option on the passed claim. Given a pre-defined set of times
+-- | Bermudan option on the input claim. Given a pre-defined set of times
 -- {t_1, t_2, .., t_N}, it allows the holder to acquire the underlying claim on at
 -- most one of these times.
+-- At each election time before maturity, the holder must `EXERCISE` the option or `POSTPONE`.
+-- At maturity, the holder must `EXERCISE` or `POSTPONE`.
```

Comment:
Not `EXERCISE` or `EXPIRE` at maturity?

Reply:
Indeed 😓 

Thanks, I unfortunately enabled auto-merge for this one, will open a separate PR for the fix

Reply:
My bad, I didn't check auto-merge was on and I assumed the reason would have been due to a CC constraint and not a typo :)